ydb/public/lib/ydb_cli/commands/ydb_yql.cpp

```diff
@@ -79,53 +75,7 @@ void TCommandYql::Parse(TConfig& config) {
 }
 
 int TCommandYql::Run(TConfig& config) {
-    if (Interactive) {
-        // run interactive cli
-
-        class TLogic : public TInteractiveCli::ILogic {
-        public:
-            TLogic(TCommandYql *base, TConfig& config)
-                : Base(base)
-                , Config(config)
-            {}
-
-            virtual bool Ready(const std::string &text) override {
-                bool done = text.find(';') != std::string::npos;
-                return done;
-            }
-
-            virtual void Run(const std::string &text, const std::string &stat) override {
-                Y_UNUSED(stat);
-                try {
-                    Base->RunCommand(Config, TString(text.c_str()));
-                } catch (TYdbErrorException &error) {
-                    Cerr << error;
-                }
-                TTerminalOutput::Print("\n");
-            }
-
-        private:
-            TCommandYql* Base;
-            TConfig& Config;
-        };
-
-
-        TTerminalOutput::Print("\033[32mYDB Interactive CLI \033[0m");
-        TTerminalOutput::Print("\033[31m(experimental, no compatibility guarantees)\033[0m\n\n");
-        TInteractiveCli::TConfig interactiveCfg;
-        interactiveCfg.Prompt = "\033[32m=> \033[0m";
-        TInteractiveCli cli(std::make_shared<TLogic>(this, config), std::move(interactiveCfg));
-        cli.Run();
-        while(true) {
-            // blocking until input
-            auto action = GetKeyboardAction();
-            cli.HandleInput(action);
-        }
-
-        return EXIT_SUCCESS;
-    } else {
-        return RunCommand(config, Script);
-    }
+    return RunCommand(config, Script);
 }
 
 int TCommandYql::RunCommand(TConfig& config, const TString &script) {
```

Comment:
Why do we need to pass "script" as argument if we have "Script" class field. Maybe we should move RunCommand code to Run?